b3lb/rest/models.py

```diff
@@ -458,49 +476,44 @@ class Parameter(models.Model):
     ]
 
     BOOLEAN_REGEX = r'^(true|false)$'
-    BOOLEAN_PARAMETER = [
-        WEBCAMS_ONLY_FOR_MODERATOR,
-        MUTE_ON_START,
-        ALLOW_MODS_TO_UNMUTE_USERS,
-        LOCK_SETTINGS_DISABLE_CAM,
-        LOCK_SETTINGS_DISABLE_MIC,
-        LOCK_SETTINGS_DISABLE_PRIVATE_CHAT,
-        LOCK_SETTINGS_DISABLE_PUBLIC_CHAT,
-        LOCK_SETTINGS_DISABLE_NOTE,
-        LOCK_SETTINGS_LOCKED_LAYOUT,
-        LOCK_SETTINGS_HIDE_USER_LIST,
-        LOCK_SETTINGS_LOCK_ON_JOIN,
-        LOCK_SETTINGS_LOCK_ON_JOIN_CONFIGURABLE,
-        MEETING_KEEP_EVENT
-    ]
-
-    NUMBER_REGEX = r'^[\d]+$'
-    NUMBER_PARAMETERS = [MAX_PARTICIPANTS, DURATION]
-
+    NUMBER_REGEX = r'^\d+$'
     POLICY_REGEX = r'^(ALWAYS_ACCEPT|ALWAYS_DENY|ASK_MODERATOR)$'
-    POLICY_PARAMETERS = [GUEST_POLICY]
-
-    COLOR_REGEX = r'^#[A-F0-9]{6}$'
-    COLOR_PARAMETER = [BANNER_COLOR]
+    COLOR_REGEX = r'^#[a-fA-F0-9]{6}$'
+    URL_REGEX = r'^(http(s)?:\/\/)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}(\.[a-z]{2,6})?\b([-a-zA-Z0-9@:%_\+.~#?&\/\/=]*)$'
+    ANY_STING_REGEX = r'^.+$'
```

Comment:
`STING` is a typo? But it's not a `string`, it might even be a number. The anchors are overhead, suggestion for a minmal non-empty value:
```python
ANY_REGEX = r'.'
```